src/php/ext/grpc/php_grpc.c

```diff
@@ -206,9 +206,8 @@ void postfork_child() {
   grpc_init();
   grpc_php_init_completion_queue(TSRMLS_C);
 
-  // re-create grpc_channel and point wrapped to it
-  // unlock wrapped grpc channel mutex
-  restart_channels();
```

Comment:
Also we probably should remove the code for the `restart_channels` function itself because no code is calling it now.

Reply:
comment addressed